src/qibo/models/circuit.py

```diff
@@ -1012,6 +1024,41 @@ def __call__(self, initial_state=None, nshots=None):
         """Equivalent to ``circuit.execute``."""
         return self.execute(initial_state=initial_state, nshots=nshots)
 
+    @property
+    def raw(self) -> dict:
```

Comment:
raw -> to_dict

Reply:
The reason why we started calling `raw` and not `to_dict` is because it is not a method but a property.

Functions (including methods) should have names of actions, but attributes (including properties) should have name of objects (what it *is* and not what it *does*).

Reply:
Ok, fine by me then.